docs/sources/project/test-and-docs.md

```diff
@@ -230,6 +230,42 @@ with new memory settings.
 6. Restart your container and try your test again.
 
 
+## Testing just the Windows client
+
+This explains how to test the Windows client on a Windows server set up as a development environment.
+
+
+1.  If you don't have one, start a Git Bash window.
```

Comment:
I think you are right, I moved this piece in and should have taken a look at that reference. I'll take care of it and resubmit.